### ISSUE TITLE
no support for multiple files

### ISSUE THREAD

**Issue by amirgee007** [JeremyFagis/dropify#94]
hi i want to use it as for uploading multiple files like name="images[]"

but i upload only first or latest file.

**Comment by tkaprol** [comment]
Create multiple upload fields as you proposed then apply dropify to all of these fields.

**Comment by amirgee007** [comment]
yes, i tried that in ajax just make dive and hide it and on button click copy this and show this.
but dropify only works for the first one not for the others that show below the first one
The design is ok but no preview available.

BTW thanks for suggestion
